src/main/java/internet/shop/Demo.java

```diff
@@ -1,121 +1,152 @@
 package internet.shop;
 
 import internet.shop.lib.Injector;
+import internet.shop.model.Car;
+import internet.shop.model.Driver;
 import internet.shop.model.Manufacturer;
+import internet.shop.service.CarService;
+import internet.shop.service.DriverService;
 import internet.shop.service.ManufacturerService;
 
 public class Demo {
     private static Injector injector = Injector.getInstance("internet.shop");
 
     public static void main(String[] args) {
-        Manufacturer manufacturer = new Manufacturer();
-        manufacturer.setId(10L);
-        manufacturer.setCountry("Ukraine");
-        manufacturer.setName("MiksMaks");
+        Driver driver1 = new Driver();
+        driver1.setId(1L);
```

Comment:
do not set the id anywhere - it will be set after you call `create()`

Reply:
done